README.md

```
@@ -0,0 +1,1368 @@
+code](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) can be used.
+
+Many countries have other categories of holidays in addition to common (national-wide) holidays:
+bank holidays, school holidays, additional (paid or non-paid) holidays, holidays of state or public
+employees, religious holidays (valid only for these religions followers). A list of all categories
+supported by country is defined by `supported_categories` (optional) attribute.
+
+The following is a list of supported countries, their subdivisions followed by their aliases (if
+any) in brackets, available languages and additional holiday categories. All countries support
+**PUBLIC** holidays category by default. All other default values are highlighted with bold:
+
+<table style="width: 100%">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: table

(MD033, no-inline-html)